packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -1244,6 +1244,15 @@ describe('lib/senders/email:', () => {
     });
   }
 
+  it('formats currency strings sanely', () => {
+    const result = mailer._getLocalizedCurrencyString(
+      123,
+      'USD',
+      'en-US;q=0.5,en;q=0.3,en-NZ'
```

Comment:
Hmm, trying to remember how these priorities work. Would have thought that the en-US q value would take priority. Seems to work whether I understand it or not, though. :)